docs/UserGuide.adoc

```diff
@@ -206,14 +199,15 @@ Examples:
 
 * *Clear* : `clear`
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
-e.g. `find punggol`
+e.g. `find punggol` or `f punggol`
 * *Filter* : `filter [f/FREE_PARKING] [n/NIGHT_PARKING] [ct/CARPARK_TYPE]` +
-e.g. `filter f/true n/false ct/multi`
-* *Help* : `help`
+e.g. `filter f/true n/false ct/multi` or `fi f/true n/false ct/multi`
+* *Help* : `help` or `h`
 * *History* : `history`
-* *List* : `list`
-* *Notify* : `notify`
-* *Sort* : `sort FILTER_TYPE` +
-e.g. `sort DISTANCE`
-* *Select* : `select CARPARK_NO` +
-e.g. `select HG83`
+* *List* : `list` or `l`
```

Comment:
I feel like shouldn't add alias under command summary. Either that or separate into two sections, one normal and the other using alias.

Reply:
Ok, I agree.